blobmodel/model.py

```diff
@@ -248,17 +245,17 @@ def integrate(self, file_name='2d_blobs.nc', speed_up=False, truncation_Lx=3):
                 start = int(b.t_init/self.dt)
                 stop = int(truncation_Lx*self.Lx/(b.v_x*self.dt)) + start
                 output[:, :, start:stop] += b.discretize_blob(
-                    x=__xx[:, :, start:stop], y=__yy[:, :, start:stop], t=__tt[:, :, start:stop])
-                if(self.periodic_y):
+                    x=__xx[:, :, start:stop], y=__yy[:, :, start:stop], t=__tt[:, :, start:stop], periodic_y=self.periodic_y, Ly=self.Ly)
+                if self.periodic_y:
```

Comment:
I think lines 249 - 253 can be deleted since they originate from the previous implementation of periodic_y

Reply:
yep, also from 256 I assume